src/Compiler/CompilerOptions.cpp

```diff
@@ -281,6 +282,19 @@ static llvm::cl::opt<std::string, true> shapeInformationOpt("shapeInformation",
     llvm::cl::value_desc("value"), llvm::cl::location(shapeInformation),
     llvm::cl::cat(OnnxMlirOptions));
 
+static llvm::cl::opt<std::string, true> dimParamsOpt("dimParams",
+    llvm::cl::desc(
+        "Custom onnx.dim_params attributes for the inputs of the ONNX model for"
+        "specifying relationship among ranks of the inputs.\n"
```

Comment:
`among ranks of` => `among dynamic dimensions of `

Reply:
Fixed. Thanks.